articles/azure-maps/how-to-view-api-usage.md

```diff
@@ -23,7 +24,7 @@ Continue to the next section if you need to customize these graphs for your part
 
 ## View detailed metrics
 
-1. Sign in to your Azure subscription in the [portal](https://portal.azure.com).
+1. Sign into the [Azure portal](https://portal.azure.com).
```

Comment:
```suggestion
1. Sign in to the [Azure portal](https://portal.azure.com).
```
In this context, "into" is not correct. See the [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2696&topicid=28035) entry.